src/containers/SubjectViewer.jsx

```diff
@@ -243,6 +263,12 @@ class SubjectViewer extends React.Component {
     this.props.dispatch(updateViewerSize(svgW, svgH));
   }
 
+  escapeCrop(e) {
+    if (e.keyCode === 27 && this.props.viewerState === SUBJECTVIEWER_STATE.CROPPING) {
```

Comment:
Can you please change `e.keyCode === 27` to `getKeyCode(e) === KEY_CODES.ESCAPE` ? (See `lib/Utility.js`)

Two reasons:
1. Using KEY_CODES.ESCAPE is somewhat more readable that simply 27.
2. [e.keyCode](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) is being deprecated, so it's good to future-proof our use of it.

Good news is, we already have a library for understanding keyboard input. Finally, all those hackdays spent refining video game controls have paid off!

(Side note: I think I saw an e.keyCode being used somewhere else too, in an earlier PR - I should check if I remembered to fix that.)